.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -15,7 +15,6 @@ body:
       description: Which component(s) does your bug report concern?
       multiple: true
       options:
-      - operator
```

Comment:
in slack we discuss to remove the area:operator label and keep labels CR/user-functionality focused.